spec/features/user_sees_homepage_spec.rb

```diff
@@ -40,7 +40,7 @@
     expect(current_path).to eq(activities_path)
   end
 
-  it 'has a cart link that works' do
+  it 'has a cart_trips link that works' do
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.